api/licensee.api

```diff
@@ -852,7 +852,6 @@ public final class app/cash/licensee/UnknownLicense$Companion {
 public final class app/cash/licensee/UnusedAction : java/lang/Enum {
 	public static final field IGNORE Lapp/cash/licensee/UnusedAction;
 	public static final field LOG Lapp/cash/licensee/UnusedAction;
-	public static fun getEntries ()Lkotlin/enums/EnumEntries;
```

Comment:
`apiVersion` didn't work as we expected. The newer language features are still enabled like this, it could be easily misused.

Reply:
This is surprising to the point where I'm wondering if it's actually a bug